network/ocsp_test.go

```diff
@@ -256,6 +259,80 @@ func TestDefaultOCSPClient(t *testing.T) {
 	checkCase(t, revokedData)
 }
 
+func testWithConfigAndValidChain(t *testing.T, ocspConfig *OCSPConfig, rawCerts [][]byte, verifiedChains [][]*x509.Certificate) {
+	ocspClient := DefaultOCSPClient{}
+
+	ocspVerifier := DefaultOCSPVerifier{Config: *ocspConfig, Client: ocspClient}
+
+	err := ocspVerifier.Verify(rawCerts, verifiedChains)
+	if err != nil {
+		t.Fatalf("Unexpected error for valid certificate: %v", err)
```

Comment:
please, add "\n" to the end of string because in the output it will be concatenated to the previous string